lms/djangoapps/grades/tests/utils.py

```diff
@@ -24,17 +24,17 @@ def mock_get_score(earned=0, possible=1):
     Mocks the get_score function to return a valid grade.
     """
     with patch('lms.djangoapps.grades.new.subsection_grade.get_score') as mock_score:
-        mock_score.return_value = ProblemScore(earned, possible, earned, possible, 1, True, None, None)
+        mock_score.return_value = ProblemScore(earned, possible, earned, possible, 1, True, None, None, True)
```

Comment:
good point.  updated.  (In this case, I agree that the vertical real-estate required to add the keyword-name helps with the readability.)